rust/osm2lanes/src/locale.rs

```diff
@@ -50,26 +46,48 @@ impl Config {
         todo!();
     }
 
-    pub fn iso_3166(self, _code: &str) -> Self {
-        todo!();
+    /// Assign ISO-3166
+    ///
+    /// Accepts any of ISO-3166-1 alpha-2,  ISO-3166-1 alpha-3, or ISO-3166-2 codes
+    pub fn iso_3166(mut self, code: &str) -> Self {
+        if code.len() == 2 {
+            self.iso_3166_1_alpha_2 = Some(code.to_owned());
+        } else if code.len() == 3 {
+            self.iso_3166_1_alpha_3 = Some(code.to_owned());
+        } else if let Some((alpha_2, subdivision)) = code.split_once('-') {
+            self.iso_3166_1_alpha_2 = Some(alpha_2.to_owned());
+            self.iso_3166_2_subdivision = Some(subdivision.to_owned());
+        }
+        self
     }
 
-    pub fn driving_side(mut self, side: DrivingSide) -> Self {
-        self.driving_side = Some(side);
+    pub fn iso_3166_option(mut self, code: &std::option::Option<std::string::String>) -> Self {
```

Comment:
```suggestion
    pub fn iso_3166_option(mut self, code: &<String>) -> Self {
```
A reason for fully qualified name?

Reply:
How does this work?

The input parameter is an option. I don't know if this will build so will test locally.

Reply:
ah, I think you didnt mean to remove the Option, but yes, my auto-refactor tool is a little broken